roxctl/central/export/deployments/deployments.go

```diff
@@ -0,0 +1,58 @@
+package deployments
+
+import (
+	"context"
+	"io"
+
+	"github.com/golang/protobuf/jsonpb"
+	"github.com/pkg/errors"
+	"github.com/spf13/cobra"
+	v1 "github.com/stackrox/rox/generated/api/v1"
+	pkgCommon "github.com/stackrox/rox/pkg/roxctl/common"
+	"github.com/stackrox/rox/pkg/utils"
+	"github.com/stackrox/rox/roxctl/central/export/common"
+	"github.com/stackrox/rox/roxctl/common/environment"
+	"github.com/stackrox/rox/roxctl/common/flags"
+)
+
+// Command defines the central command tree
+func Command(cliEnvironment environment.Environment) *cobra.Command {
+	c := &cobra.Command{
+		Use:   "deployments",
+		Short: "Commands related to exporting deployments from Central.",
```

Comment:
This is actually doing the export so short should be:
```suggestion
		Short: "Exports all deployments from Central.",
```